Config/Pipeline/config_WP3.yaml

```diff
@@ -3,7 +3,7 @@ programs:
     Trimming: "Fastp" #Cutadapt / Fastp / None
     markduplicate: "CPU" #GPU / CPU
 
-bam_extra: "-c 250 -M "
+bam_extra: " -c 250 -M "
```

Comment:
Is space needed? Looks like it should be fine without it 

https://github.com/clinical-genomics-uppsala/Twist_DNA/blob/21ec9195e8131d437ef3f1d7fbb7ebe03d313512/src/Snakemake/rules/Alignment/bwa-mem.smk#L74